doc/source/whatsnew/v0.25.0.rst

```diff
@@ -70,12 +70,27 @@ is respected in indexing. (:issue:`24076`, :issue:`16785`)
 Increased minimum versions for dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We have updated our minimum supported versions of dependencies (:issue:`23519`).
+Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions.
+Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`24942`).
 If installed, we now require:
 
 +-----------------+-----------------+----------+
 | Package         | Minimum Version | Required |
 +=================+=================+==========+
+| beautifulsoup4  | 4.4.1           |          |
++-----------------+-----------------+----------+
+| openpyxl        | 2.2.6           |          |
```

Comment:
are these the minimums implied by the py3 builds?

Reply:
@h-vetinari ?

Reply:
Since you asked me to remove the compat jobs, all of the libraries here are unpinned for the rest of the remaining CI. They are however the minimum of what conda will resolve for PY35, so these are the implied minimums as you say.

I will re-add another PR with all the work that was done here on the PY35 jobs anyway.

Reply:
ok thanks. pls do so.